app/controllers/api/v1/auth_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  module V1
+    class AuthController < ApplicationController
+      before_action :authenticate, only: :logout
+
+      def login
+        parameters = {
+          fields: FIELDS,
+          access_token: auth_params[:access_token]
+        }
+
+        http_client = Http.new(FB_URL)
+        response, response_status = http_client.get_request(parameters)
+        message, status =  authenticate_user(response, response_status)
+        render json: message, status: status
+      end
+
+      def logout
+        ExpiredToken.create(token: token)
```

Comment:
What do you think of saving active tokens instead of expired ones and deleting them on log out?